ci/init-plugin-test-service.sh

```diff
@@ -41,3 +41,70 @@ docker exec -i rmqnamesrv /home/rocketmq/rocketmq-4.6.0/bin/mqadmin updateTopic
 
 # prepare vault kv engine
 docker exec -i vault sh -c "VAULT_TOKEN='root' VAULT_ADDR='http://0.0.0.0:8200' vault secrets enable -path=kv -version=1 kv"
+
+# prepare openfunction env
+prepare_kind_k8s() {
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
+    chmod +x ./kind
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+    sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+    kind create cluster --name myk8s-01
+}
+
+install_openfuncntion() {
+    curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+    helm repo add openfunction https://openfunction.github.io/charts/
+    helm repo update
+    kubectl create namespace openfunction
+    helm install openfunction --set global.Keda.enabled=false --set global.Dapr.enabled=false openfunction/openfunction -n openfunction
+    kubectl wait pods --all  --for=condition=Ready --timeout=300s -n openfunction
+    kubectl delete -A ValidatingWebhookConfiguration ingress-nginx-admission
+    kubectl delete deployment -n keda --all
+    kubectl delete deployment -n dapr-system --all
+    kubectl delete pod -n keda --all
+    kubectl delete pod -n dapr-system --all
+}
+
+set_container_registry_secret() {
+    REGISTRY_SERVER=https://index.docker.io/v1/ REGISTRY_USER=apisixtestaccount123 REGISTRY_PASSWORD=apisixtestaccount
```

Comment:
Why use this? We can push the docker images through `kind load docker-image`.

Reply:
The part of the openfuction building lifecycle, pushing the image to a registry, needs it.Will it be better to build locally to avoid this part?

Reply:
You should avoid exposing your secrets here. By the way, is this for building openfunction itself or the function?

Reply:
Sure，for the function.
I 'll avoid it.

Reply:
Is there a lighter way (using less runtime memory) to run these services? I'm worried that running these services will again fill up the memory of the server where github action is running CI.

Reply:
My plan is  to just run the function locally in docker with the image I created by openfunction platform previously，which doesn't need to install many services. I'll update it after test.